src/main/java/com/ufcg/sad/models/questao/Questao.java

```diff
@@ -17,12 +17,13 @@
  * @author Lucas Silva, Marianne Linhares
  */
 @Entity
+@Table
```

Comment:
Por que o adicionou essa anotação?

Reply:
Só coloquei por padronização, todas as outras entidades usam @Table. Mas só essa anotação sem parâmetro não muda nada.

"@Table maps the entity with the table. If no @Table is defined, the default value is used: the class name of the entity."